src/server/game/Entities/Player/Player.cpp

```diff
@@ -1563,10 +1563,15 @@ void Player::SetDrunkValue(uint8 newDrunkValue, uint32 itemId /*= 0*/)
     if (newDrunkenState == oldDrunkenState)
         return;
 
-    WorldPacket data(SMSG_CROSSED_INEBRIATION_THRESHOLD, (8+4+4));
-    data << uint64(GetGUID());
-    data << uint32(newDrunkenState);
+    ObjectGuid guid = GetGUID();
+    WorldPacket data(SMSG_CROSSED_INEBRIATION_THRESHOLD, (8 + 4 + 4));
+    data.WriteGuidMask(guid, 0, 4, 2, 6, 5, 1, 3, 7);
+
+    data.WriteByteSeq(guid[3]);

```

Comment:
change to 
`data.WriteGuidBytes(guid, 3);` 
? (Code Style :P)
also packet doesn't need FlushBits?

Reply:
Code style fixed, FlushBits is not required when you write the whole byte (anything divisible by 8) which happens mostly when there is just guid in the bit section.

Reply:
oh, didn't know that because client seems to flushbits always when writing with a bit https://gist.github.com/Artamedes/20bdf74575c86671941138ffa6bf1297#file-18414-cmsg_grant_level-L28